### ISSUE TITLE
Need to consider Scenario as TestCase

### ISSUE THREAD

**Issue by lsantos10** [cube-io/teamcity-formatter#4]
This will write every step from Scenario as a test case which is wrong. Need to consider the feature as test suite and scenario as Test Case.

**Comment by ath88** [comment]
I agree with you. If you make a pull request, I'll be willing to look it over and merge. :)
